src/calendar/grid/index.tsx

```diff
@@ -39,7 +39,7 @@ export interface GridProps {
   onFocusDate: (date: null | Date) => void;
   onChangeMonth: (date: Date) => void;
   startOfWeek: DayIndex;
-  todayAriaLabel: string;
+  todayAriaLabel?: string;
```

Comment:
if we're marking this optional, presumably we should use the hook here https://github.com/cloudscape-design/components/pull/875/files#diff-c717613d99792450f8a0070b5630d6bbf3934cbe387130bc8861e5b20b87a946R159 rather than just in the date-picker?